fs.go

```diff
@@ -80,8 +83,8 @@ func PathKind(path string) string {
 		return "http"
 	} else if strings.HasPrefix(path, "/ipfs") {
 		return "ipfs"
-	} else if strings.HasPrefix(path, "/map") || strings.HasPrefix(path, "/cafs") {
-		return "cafs"
+	} else if strings.HasPrefix(path, "/map") {
```

Comment:
Some tests in `qri` also use a "/mem" path. I have no idea if it's possible for those tests to hit this codepath, but it might be possible with future code changes? Might be worth checking for that as well.